src/devices/cpu/h8/h8_sci.h

```diff
@@ -63,14 +63,14 @@ class h8_sci_device : public device_t {
 		CLK_RX = 2
 	};
 
-	enum {
-		CLKM_INTERNAL_ASYNC,
-		CLKM_INTERNAL_ASYNC_OUT,
-		CLKM_EXTERNAL_ASYNC,
-		CLKM_EXTERNAL_RATE_ASYNC,
-		CLKM_INTERNAL_SYNC_OUT,
-		CLKM_EXTERNAL_SYNC,
-		CLKM_EXTERNAL_RATE_SYNC
+	enum class ClockMode {
+		INTERNAL_ASYNC,
+		INTERNAL_ASYNC_OUT,
+		EXTERNAL_ASYNC,
+		EXTERNAL_RATE_ASYNC,
+		INTERNAL_SYNC_OUT,
+		EXTERNAL_SYNC,
+		EXTERNAL_RATE_SYNC
 	};
```

Comment:
We use `snake_case` for type names.  `LlamaCase` is used for template parameter names.  See here: https://docs.mamedev.org/contributing/cxx.html#naming-conventions

Reply:
I've pushed an update. As it stands the enum typename and the var are the exact same name, which feels like a bad idea? Any thoughts, or name suggestions?